File: YZDEV20161107-kevin.wen-a0ce459eeba41e012f375d8de/YZDEV20161107-kevin.wen-a0ce459eeba41e012f375d8dee60c377131493f1/mvc/src/com/augmentum/oes/service/UserService.java
package com.augmentum.oes.service;

import com.augmentum.oes.exception.ParamterException;
import com.augmentum.oes.exception.ServiceException;
import com.augmentum.oes.model.User;

public interface UserService {

    public User login(String userName,String password) throws ParamterException,ServiceException;

}

File: YZDEV20161107-dotnet_kevin-6ccf4b83ce002b6bcf2bb1831aed4c0b6f58f82e/oes/OESWCF/OESLogic/UserService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using OESDal;
using OESModel;
using OESUtil;
using OESCommon;
using System.ServiceModel;

namespace OESLogic
{
    public class UserService : IUserService
    {
        public bool verifyUserLogin(string userName, string password)
        {
            UserDal userDal = new UserDal();
            User user = userDal.Login(userName);
            if (user == null) 
            {
                throw new FaultException(Constants.UserNoExit);
            }

            if (user.Password == password)
            {
                SessionUtil.User = user;
                return true;
            }
            else
            {
                return false;
            }
            
        }
    }
}
